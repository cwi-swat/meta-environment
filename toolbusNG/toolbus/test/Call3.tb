process P(N : int, M : int) is Q(N, M?) . print N . print M

process Q(N : int, M : int?) is N := mul(N,2) . M := mul(M, 2) . R(N, M?) . print N . print M

process R(N : int, M : int?) is N := mul(N,3) . M := mul(M,3) . print N . print M

toolbus(P(5, 100))
