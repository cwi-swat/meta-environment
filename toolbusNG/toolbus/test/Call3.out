30
600
10
600
5
600
